src/aqua_diagnostics/core/util.py

```diff
@@ -174,3 +173,43 @@ def merge_config_args(config: dict, args: argparse.Namespace,
             logger.debug(f"  - {ref['catalog']} {ref['model']} {ref['exp']} {ref['source']}")
 
     return config
+
```

Comment:
this function is not necessary: a proper common fucntion has been introduced in `aqua.util`, please update your code from the main and use this one to save lines! Thanks!